docs/src/developers_guide/contributing_documentation_easy.rst

```diff
@@ -0,0 +1,95 @@
+
+Documentation is important and we encourage any improvements that can be made.
+If you believe the documentation is not clear please contribute a change to
+improve the documentation for all users.
+
+The guide below is designed to be accessible to those with little-to-no
+knowledge of programming and GitHub. If you find that something doesn't work as
+described or could use more explanation then please let us know (or contribute
+the improvement yourself)!
+
+First time only steps
+^^^^^^^^^^^^^^^^^^^^^
+ 
+1. Create a `GitHub <https://github.com/>`_ account.
+
+2. Complete the Scitools Contributor License Agreement (`link to Google Form
+   <https://docs.google.com/forms/d/e/1FAIpQLSfd0tdE-DcJOXh8ej_7T93IizwJFYBFyRWYQOi2A8QRaKwykA/viewform>`_).
+   This is a one-off requirement for anyone who wishes to contribute to a
+   Scitools repository - including the documentation.
+
+Steps to complete each time you propose changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Navigate to documentation page that you want to edit (on this site).
+
+2. Click the 'Edit on GitHub' button at the top right of the page.
+
+.. image:: edit_on_github.png
+
+3. In the resulting GitHub page select **main** from the drop-down menu near
+   the top left of the page (to the left of the 'iris / docs / src / …' links)
+   if it isn't already. This changes the branch to main.
+
+.. image:: find_main.png
+
+4. Click the pencil symbol near the top right (to the right of the 'Raw' and
+   'Blame' buttons).
+
+.. image:: edit_button.png
+
+5. Make your edits! Try to strike a balance between informing the audience
+   enough that they understand and overwhelming them with information.
+
+.. note::
+
+    You may see the following message at the top of the edit page, informing you
+    that GitHub has created you your own 'fork' (or copy) of the project as a
+    precursor to allowing you to edit the page. Your changes will be merged into
+    the main version of the documentation later.
+
+    .. image:: fork_banner.png
+
+6. Scroll to the bottom of the edit page and enter some appropriate information
+   in the two boxes under 'Propose changes'. You can just keep the default text
+   if you like or enter something more specific - a short sentence explaining
+   what's changed is fine. Then click the 'Propose changes' button.
+
+.. image:: propose_changes.png
+
+7. In the resulting page titled 'Pull Request', write a brief description of
+   what you've changed underneath the following three lines:
+
+.. code::
+
+    ### Description
+    <!-- Provide a clear description about your awesome pull request -->
+    <!-- Tell us all about your new feature, improvement, or bug fix -->
+
+Describing what you've changed and why will help the person who reviews your changes.
+
+.. image:: pull_request.png
+
+8.	Click the 'Create pull request' button.
+
+What happens next?
+^^^^^^^^^^^^^^^^^^
+
+Someone else who works on Iris will review your changes (this happens for
```

Comment:
`Someone else who works on Iris` -> `Another Iris contributor`

For me `contributor` is more open-ended, as anyone can add a comment for review, not just a core developer... that's what I'm trying to consciously communicate here.

Reply:
I love how this change makes them one of us, and makes it clear that contributor covers a variety of circumstances